ocaml/xapi/xapi_pool_helpers.ml

```diff
@@ -269,3 +269,221 @@ let apply_guest_agent_config ~__context =
         (Printexc.to_string e)
   in
   call_fn_on_slaves_then_master ~__context f
+
+module PeriodicUpdateSync = struct
+  let periodic_update_sync_task_name = "Periodic update synchronization"
+
+  exception UpdateSync_RetryNumExceeded of int
+
+  let seconds_random_within_a_day () =
+    if Xapi_fist.disable_periodic_update_sync_sec_randomness () then
+      0.
+    else
+      let secs_of_a_day = 24. *. 60. *. 60. in
+      let secs_random = Random.float secs_of_a_day in
+      secs_random
+
+  let frequency_to_str ~frequency =
+    match frequency with
+    | `daily ->
+        "daily"
+    | `monthly ->
```

Comment:
No `monthly` any more?

Reply:
No, `monthly` is not supported now.
Sorry, I missed to remove the code here, but it is strange that the compile did not fail.